oauth-keycloak-authorizer/src/main/java/io/strimzi/kafka/oauth/server/authorizer/KeycloakAuthorizer.java

```diff
@@ -85,13 +89,74 @@ public void configure(Map<String, ?> configs) {
         if (configuration.isDelegateToKafkaACL()) {
             delegate = instantiateStandardAuthorizer();
             delegate.configure(configs);
+            initPluginMetricsIfNeeded();
         }
 
         if (log.isDebugEnabled()) {
             log.debug("Configured " + this + " using " + singleton);
         }
     }
 
+    /**
+     * Initialise and set PluginMetrics on StandardAuthorizer if it implements Monitorable
+     * Use of reflection is needed since the classes are only available starting with Kafka 4.1.0
+     * Once this class implements Monitorable the use of reflection will no longer be necessary, and the passed-in
+     * PluginMetrics object will be passed on to StandardAuthorizer.withPluginMetrics().
+     */
+    private void initPluginMetricsIfNeeded() {
+        Class<?> monitorableClass = null;
+        try {
+            // Check if StandardAuthorizer implements Monitorable, which can only be true if running on Kafka 4.1.0+
```

Comment:
The comment is a bit misleading. This block checks if the Monitorable class is present, ie, we're running >=4.1

Reply:
What about `// Check if Monitorable is present. This indicates if we're running on Kafka 4.1.0+`?